### PR TITLE
2930: Non-normative changes to variable names to match 1559.

### PR DESCRIPTION
**Pull request by MicahZoltu**
There has been some desire to get some consistency across these EIPs in terms of naming.  Also, `access_list` is snake case while the rest are `camelCase` so I fixed it to align with the rest (and also match casing style of 1559).

### CODE REVIEW

**Comment by MicahZoltu** [comment]
cc @axic per your request.  😄

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - eip-2930.md is in state final, not draft or last call or review
	 - eip-2930.md requires approval from one of (vbuterin, holiman)
 @vbuterin @holiman

**Comment by holiman** [comment]
I approve

**Comment by MicahZoltu** [comment]
@holiman I need you to click the GitHub approve button or else the bot blocks merging.  😢

**Comment by holiman** [comment]
Ok, will do when by computer

**Review by holiman — APPROVED** [review]

**Pull request merged**
